lib/pluggy_elixir/error/unauthorized.ex

```diff
@@ -0,0 +1,11 @@
+defmodule PluggyElixir.Error.Unauthorized do
+  @moduledoc """
+  Error for invalid credentials.
+  """
+
+  @type t :: %__MODULE__{
+          message: binary()
+        }
+
+  defstruct [:message]
```

Comment:
According to [Elixir Style Guide](https://github.com/christopheradams/elixir_style_guide#module-attribute-ordering) `defstruct` should came before `@type`.

Besides that, should we add `@enforce_keys [:message]` to the `defstruct`?

Reply:
👍🏻